assets/src/domain/eventEditor/ui/ticketAssignmentsManager/RenderTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Table } from 'antd';
+
+import { RenderTableProps } from './types';
+import { Datetime } from '@edtrServices/apollo/types';
+import RenderCell from './RenderCell';
+import useRowClassName from './useRowClassName';
+import useColClassName from './useColClassName';
+import useCellProps from './useCellProps';
+
+const { Column } = Table;
+
+const RenderTable: React.FC<RenderTableProps> = ({ datetimes, tickets }) => {
+	const getRowClass = useRowClassName();
+	const getColClass = useColClassName();
+	const getCellProps = useCellProps();
+
+	return (
+		<Table<Datetime>
+			dataSource={datetimes}
+			rowKey={(item) => item.id}
+			pagination={{ pageSize: 5 }}
+			rowClassName={(datetime) => getRowClass(datetime)}
+			id='ticket-assignment-manager'
+			bordered
+		>
+			<Column<Datetime> title={null} key={'titlecol'} render={(_, datetime) => <span>{datetime.name}</span>} />
```

Comment:
ok what is this voodoo? Since when can you just directly pass TS types into a react component like that?

Reply:
Support for **Generic type arguments in JSX elements** was added in TS 2.9
[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html)

Reply:
cool, thanks for the link, I had not seen that in the regular docs